test/functional/notes_controller_test.rb

```diff
@@ -444,7 +444,7 @@ def teardown
     assert_response :success
     assert_not_nil @response.body
     json = JSON.parse(@response.body)
-    assert_equal ["can't be blank", "is too short (minimum is 3 characters)", "can only include letters, numbers, and dashes"], json['title']
+    assert_equal ["can't be blank", "is too short (minimum is 2 characters)", "can only include letters, numbers, and dashes"], json['title']
```

Comment:
The changes in #10809 and this one has to be in one PR since they resolve the same thing.

Reply:
Alright, I'll do it!